webapi/src/main/scala/org/knora/webapi/settings/KnoraSettings.scala

```diff
@@ -258,4 +329,37 @@ object KnoraSettings extends ExtensionId[KnoraSettingsImpl] with ExtensionIdProv
      * Java API: retrieve the Settings extension for the given system.
      */
     override def get(system: ActorSystem): KnoraSettingsImpl = super.get(system)
+
+    val featureTogglesPath: String = "app.feature-toggles"
+    val descriptionKey: String = "description"
+    val availableVersionsKey: String = "available-versions"
+    val developerEmailsKey: String = "developer-emails"
+    val expirationDateKey: String = "expiration-date"
+    val enabledByDefaultKey: String = "enabled-by-default"
+    val defaultVersionKey: String = "default-version"
+    val overrideAllowedKey: String = "override-allowed"
+
+    /**
+     * Represents the base configuration of a feature toggle.
+     *
+     * @param featureName       the name of the feature.
+     * @param description       a description of the feature.
+     * @param availableVersions the available versions of the feature.
+     * @param defaultVersion    the version of the feature that should be enabled by default.
+     * @param enabledByDefault  `true` if the feature should be enabled by default, `false` if it should be
+     *                          disabled by default.
+     * @param overrideAllowed   `true` if this configuration can be overridden, e.g. by per-request feature
+     *                          toggle configuration.
+     * @param expirationDate    the expiration date of the feature.
+     * @param developerEmails   one or more email addresses of developers who can be contacted about the feature.
```

Comment:
I am not sure why we need the developer's email? since feature toggles last a year, maybe until it is removed a developer quits, then his email address would be a useless piece of information.

Reply:
I think it's useful to know who to contact if you have questions about a feature. If a developer quits, we can update the email address.